_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,171 @@
+title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 24-25th, 2024 at Arizona State University"
+style="float: right; height: 20em; " />
+
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: right; height: 5em; " />
+
+MDAnalysis, in conjunction with the [Molecular Sciences Software Institute][]
+(MolSSI) and with the support of the [Chan-Zuckerberg Inititative][CZI] and the
+[Center for Biological Physics][CBP], is holding a **2-day** workshop on **June
+24th-25th** 2024 at **Arizona State University** in Tempe, Arizona, USA.
+
+This two-day workshop is intended to transform attendees from users to
+developers and will cover an introduction to MDAnalysis, software best
+practices, and guide participants through the creation of their own [MDAKit][mdakits].
+
+### Workshop outline
+
+MDAnalysis developers and experienced MolSSI instructors will teach
+participants how to build their own MDAnalysis-based Python packages by guiding
+them through the development of their own MDAKits. Specifically, the workshop
+will include 3 modules: (1) an introduction to using and writing custom
+analyses in MDAnalysis; (2) an overview of software development and maintenance
+best practices; and (3) an interactive hackathon session where participants
+write their own MDAKits.
+
+The program will run from the morning of Monday, June 24th until the early
+evening of Tuesday June 25th. 
+
+Parts of the workshop will be held in a hybrid mode for those unable to attend
+in-person. Registration for the hybrid components is required. Recordings of
+hybrid sessions will be made available after the workshop.
+
+Breakfast and lunch will will be provided together with tea/coffee during
+breaks.
+
+
+
+### Registration
+
+Attendance at this workshop will be *free* but limited to 40 in-person
+participants.
+
+A number of **bursaries** will be available to assist with travel and
+accommodation costs for participants from outside the Phoenix area. Bursaries
+(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
+accommodation* (single room within 5 minutes of the workshop venue) will be
+awarded.
+
+#### Application *with bursary request* (non-local attendees)
+
+Register by **May 27, 2024** (midnight anywhere on Earth).
+
+We especially welcome your application if you consider yourself being part of
+an under-represented demographic in the computational molecular sciences.
+
+Bursaries are reserved to enable the attendance of participants from outside
+the greater Phoenix area.
+
+Note that airfare for travel awardees will have to be *booked by June 3 through
+our travel agency* so as an awardee you will have to be able to
+respond to our communications (via email) immediately and act quickly.
+
+Bursary recipients will be notified by May 29. 
+
+
+#### Application without bursary request or local attendees
+
+Register by **June 5, 2024** (midnight anywhere on Earth).
```

Comment:
We want to tell non-local applicants ASAP if they have been accepted so that they can plan travel (see above). If we wait until June 5+X to tell them, they have less than 2 weeks. The alternative is to tell _all_ non-locals to apply for a bursary, regardless of if they need it, which will add even more to our reviewing load.

We could say that application deadline for non-locals is May 27. However, then we effectively give priority to non-locals as we don't know how many locals want to come.

Therefore, I suggest that we **keep a single deadline for all applications** and we make this deadline **May 27 for everyone**. In this way we will have all information when we need it.

*If we don't get enough applications we can then extend it until June 5*.

Opinions @IAlibay @fiona-naughton @jennaswa @armcdona  ?

Reply:
I agree that we should just keep it simple and have one deadline.